### DISCUSSION TITLE
Textarea with 2 different views

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#4025] [Q&A]
Hello,

I'm using Blazorise inside a project, where a I have a backend for text recognition and now I would like to open the user interface for editing the text and afterwards parse and recognise several text passages. So I'm having the question, if there is any component in blazorise, where you can provide a "DisplayText" property and a "EditText" property?

For example:
DisplayText would be 
`mark>Sample Artist 1</mark> - <mark>Sample Title 1</mark>				00:00:00`

which will be rendered to display the recognized text as highlight.
EditText would be without the mark logic: 

`Sample Artist 1 - Sample Title 1				00:00:00`

User can only edit the "EditText" and the DisplayText would get set from the backend. Is there anything that works like this?

**Comment by stsrki** [comment]
The closest to a component with a preview is the [Markdown](https://blazorise.com/docs/extensions/markdown) component.

Other than that you could build a component that has two TextEdits and a button to hide and show them.

**Reply by NeoCoderMatrix86** [reply]
Ok, thanks for your answer. I played around a bit, but I'm currently a bit confused, how to get a Markdown string into the component in order to interpret it (Highlight several text areas). Are there any examples?
